test/boolean_test.cpp

```diff
@@ -138,6 +138,14 @@ TEST(Boolean, Cubes) {
 #endif
 }
 
+TEST(Boolean, Simplify) {
+  Manifold cube = Manifold::Cube().Refine(10);
+  Manifold result = cube + cube.Translate({1, 0, 0});
+  EXPECT_EQ(result.NumTri(), 2000);
```

Comment:
I need to figure out why I'm not getting the number of faces I expect here, but it's late, so I'll leave this for tonight.